pages/[sectionSlug]/[articleSlug].vue

```diff
@@ -22,11 +23,17 @@ if (article.leadGallery) {
 const topImage = article.leadImage || gallery.slides[0].image
 const topCaption = article.leadImageCaption || gallery?.slides[0].image.title
 const galleryLength = gallery?.slides.length || 0
-
+const config = useRuntimeConfig()
 const trackingData = useArticlePageTrackingData(article)
 const adTargetingData = useArticlePageAdTargetingData(article)
 const sensitiveContent = useSensitiveContent()
 const headMetadata = useArticlePageHeadMetadata(article)
+const { y: scrollY } = useWindowScroll()
+const isMediumOrUpScreen = useMediaQuery(`(min-width: 768px)`)
+// maybe these values should be calculated automatically by measuring on the position of the main header
```

Comment:
we can use gsap scrollTrigger in the future to determine when the regular header leaves the viewport.